pyiron_base/jobs/job/generic.py

```diff
@@ -1087,17 +1093,9 @@ def to_hdf(self, hdf=None, group_name=None):
             hdf (ProjectHDFio): HDF5 group object - optional
             group_name (str): HDF5 subgroup name - optional
         """
-
         self._set_hdf(hdf=hdf, group_name=group_name)
-        self._executable_activate_mpi()
```

Comment:
Seem unrelated to the other changes, is that generally not necessary anymore?

Reply:
The function was just moved to line 1053.